contributing/topics/reference-azure-resource-ids.md

```diff
@@ -0,0 +1,3 @@
+# Azure Resource IDs
+
+*This section not yet complete*
```

Comment:
do we want to exclude this for now?

Reply:
Sure, I'll remove it along with the links to it 👍